ingest-app/src/cmr/ingest/validation/validation.clj

```diff
@@ -156,17 +158,32 @@
 
 (defn validate-granule-umm
   "Validates a UMM granule record using rules defined in UMM with a UMM collection record,
-   updated with platform aliases whoes shortnames don't exist in the platforms."
+  updated with platform aliases whoes shortnames don't exist in the platforms."
   [context collection granule]
   (if-errors-throw :invalid-data (umm-validation/validate-granule
-                                  (humanizer-alias-cache/update-collection-with-aliases context
-                                                                                        collection
-                                                                                        false)
+                                  (humanizer-alias-cache/update-collection-with-aliases
+                                   context collection false)
                                   granule)))
 
+(defn validate-variable-umm-spec-schema
+  "Validate the variable against the JSON schema and throw errors if
+  configured or return a list of warnings"
+  [variable validation-options]
+  (if-let [err-messages (seq (json-schema/validate-umm-json
+                              (umm-json/umm->json variable)
+                              :variable))]
+    (if (or (:validate-umm? validation-options)
+            (config/return-umm-json-validation-errors))
```

Comment:
This was intended for UMM-C only. We'll need to introduce a separate config if we want to do something similar for variables. I don't think we have any need to do this though so we should remove the idea of warnings for variables.